scopesim/optics/image_plane_utils.py

```diff
@@ -32,29 +32,29 @@ def get_canvas_header(hdu_or_table_list, pixel_scale=1 * u.arcsec):
 
     """
 
-    size_warning = "Header dimension are {} large: {}. Any image made from " \
-                   "this header will use more that >{} in memory"
+    size_warning = ("Header dimension are {adverb} large: {num_pix}. Any image "
+                    "made from this header will use more that >{size} in memory")
 
     headers = [ht.header for ht in hdu_or_table_list
                if isinstance(ht, fits.ImageHDU)]
-    if sum([isinstance(ht, Table) for ht in hdu_or_table_list]) > 0:
+    if sum(isinstance(ht, Table) for ht in hdu_or_table_list):
```

Comment:
```suggestion
    if any(isinstance(ht, Table) for ht in hdu_or_table_list):
```
That would be the same thing right?